src/views/viewer/ChannelsView/ChannelsView.tsx

```diff
@@ -1,15 +1,19 @@
 import styled from '@emotion/styled'
 import React from 'react'
 
+import { useMostFollowedChannelsAllTimeIds } from '@/api/hooks'
 import { InfiniteChannelWithVideosGrid, ViewWrapper } from '@/components'
 import { Text } from '@/shared/components'
 import { sizes } from '@/shared/theme'
 
 export const ChannelsView = () => {
+  const { mostFollowedChannelsAllTime } = useMostFollowedChannelsAllTimeIds({ limit: 15 })
+  const mostFollowedChannelsAllTimeIds = mostFollowedChannelsAllTime?.map((item) => item.id)
   return (
     <StyledViewWrapper>
       <Header variant="h2">Browse channels</Header>
-      <InfiniteChannelWithVideosGrid title="Channels in your language:" languageSelector onDemand />
+      <StyledInfiniteChannelWithVideosGrid title="Discover channels" onDemand idIn={mostFollowedChannelsAllTimeIds} />
```

Comment:
It's not sorted by followers. When you pass the `id_in` argument to the query it will again sort by date of creation if I'm not mistaken. I had some struggles today with this problem too when I was working on promising channels and I don't really like how I resolved that.

Reply:
I'm affraid we are not able to sort anything by followers or views because of merging orion and query node data. @kdembler correct me please if I'm wrong.

Reply:
You're right that we cannot ask the query node to do the sort for us, but we can get the sorted IDs from Orion, then fetch the videos from the query node and then sort videos using IDs locally